app/views/webhooks/index.html.erb

```diff
@@ -147,10 +147,10 @@
     Outbound Webhooks
     <%= additional_info "POST to an external url after a deploy has finished. The request body will contain deploy, stage, and project information." %>
   </h2>
-  <% if @project.deploys.last %>
+  <% if @project.deploys.first %>
```

Comment:
please do this and then reuse `deploy`:
```suggestion
  <% if deploy = @project.deploys.first %>
```

Reply:
took suggestion and clarified it as latest deploy